package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "styled-components": "^3.4.2"
```

Comment:
I think this would need to be on the package.json file inside the todo directory?